imap_processing/codice/L0/decom_python_example.py

```diff
@@ -0,0 +1,218 @@
+"""
+This script contains functions to extract and decommute parameters from a binary file containing packets.
+This is a hard coded version of the decom.py file. It is used to test the decom.py file.
```

Comment:
What do you mean by "It is used to test the decom.py file?" Does that mean this file should NOT be used anywhere except in the tests? I don't see it being used anywhere.